common/src/main/java/com/netflix/conductor/common/metadata/workflow/WorkflowTask.java

```diff
@@ -160,6 +160,12 @@ public void setTasks(List<WorkflowTask> tasks) {
 	@ProtoField(id = 23)
 	private Boolean asyncComplete = false;
 
+	@ProtoField(id = 24)
+	private String loopCondition;
+
+	@ProtoField(id = 25)
+	private List<WorkflowTask> loopOver = new LinkedList<>();
```

Comment:
I guess this file is auto generated using protocol buffers. We should not manually update this.